### PR TITLE
Rewrote abstract  - ONLY REVIEW "index.src.html"

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: index.src.html [hunk 1]
```diff
@@ -471,9 +471,11 @@ <h2 class="no-toc no-num">
       Abstract
     </h2>
     <p>
-      The <code><a href="#dfn-picture">picture</a></code> element is an image
-      container whose source content is determined by one or more CSS media
-      queries.
+      The <code>picture</code> element provides a way to group multiple
+      versions of an image based on different characteristics (e.g., format,
```
Comment (anselmh) [inline index.src.html:475]: no comma after e.g.

**Comment by anselmh** [comment]
- In general looks good to me
- Noted one typo
- Should we write 'user agent' instead of browser?

File: index.src.html (hunk 1 above)
Comment (marcoscaceres) [inline index.src.html:475]: http://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e

Comment (anselmh) [inline index.src.html:475]: Oh wow, thanks! Didn't know that.

**Comment by marcoscaceres** [comment]
@anselmh thanks for the speedy review. I dropped "browser", used user agent.

**Comment by anselmh** [comment]
@marcoscaceres you're welcome. Btw: How do we handle pull-request here in general? Is one review enough? Auto-merge or manual merge with rebase? :)

**Comment by marcoscaceres** [comment]
@anselmh one review is enough for small changes. Large normative changes I like to have a few reviews. Then merge and rebase. Feel free to make pull requests :)

**Pull request merged**